pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -24,6 +24,47 @@ import { Terminal as Term } from "xterm";
 import { ContextMenu } from "cockpit-components-context-menu.jsx";
 import "console.css";
 
+const theme_core = {
+    yellow: "#b58900",
+    brightRed: "#cb4b16",
+    red: "#dc322f",
+    magenta: "#d33682",
+    brightMagenta: "#6c71c4",
+    blue: "#268bd2",
+    cyan: "#2aa198",
+    green: "#859900"
+};
+
+const dark_theme = Object.assign({
```

Comment:
`theme_core` cannot be first, because first argument is target, that is overwritten. So the last overwrite is used. I guess making deep copy would solve this problem.

Reply:
The usual way to do this is `Object.assign({}, theme_core, { overrides .. })`. But no need to  re-push again for this, we can change this later when we need it.

Reply:
I will update this as I will re-push anyway